src/storage/resourceLoader.ts

```diff
@@ -322,11 +357,15 @@ export class CCloudResourceLoader extends ResourceLoader {
     }
   }
 
+  /**
+   * Get all of the known schema registries in the accessible CCloud environments.
+   *
+   * Ensures that the coarse resources are loaded before returning the schema registries from
+   * the resource manager cache.
+   *
+   * If there is no CCloud auth session, returns an empty array.
```

Comment:
Should the function have this at the beginning? Looks like it was removed but the comment suggests otherwise
```ts
    if (!hasCCloudAuthSession()) {
      return [];
    }
```

Reply:
No, was a mistake to have added it in previously.